awx.yaml

```diff
@@ -0,0 +1,89 @@
+package:
+  name: awx
+  version: 24.6.1
+  epoch: 0
+  description: AWX provides a web-based user interface, REST API, and task engine built on top of Ansible
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - python-3.11
+
+# To do- add py-twilio and py-zstd and hiredis-py
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - bash
+      - make
+      - ca-certificates-bundle
+      - nodejs
+      - npm
+      - nvm
+      - py3-sqlparse
+      - py3.11-pip
+      - python-3.11-dev
+      - zlib-dev
+      - py3-aiohttp
```

Comment:
If these are explicity py3.11 use py3.11-package throughout to prevent default python be installed

Any runtime python deps we have should be in runtime as well as here.